{"uuid": "375eca19-3b41-4a4a-960a-4dc0042cb681", "children": ["a4b74133-a144-4791-b7c2-30bf5eee1ba7"], "befores": [{"name": "media_set_up", "status": "broken", "statusDetails": {"message": "requests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='test.hkciot.com', port=443): Max retries exceeded with url: /cuteview/media/getInPage (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x000002707FF6C910>, 'Connection to test.hkciot.com timed out. (connect timeout=None)'))\n", "trace": "  File \"D:\\sw_install\\python-install\\lib\\site-packages\\pluggy\\_callers.py\", line 77, in _multicall\n    res = hook_impl.function(*args)\n  File \"D:\\sw_install\\python-install\\lib\\site-packages\\_pytest\\fixtures.py\", line 1130, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"D:\\sw_install\\python-install\\lib\\site-packages\\_pytest\\fixtures.py\", line 909, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"D:\\ui\\test_case\\test_media_center.py\", line 24, in media_set_up\n    media_list.delete_all_media()\n  File \"D:\\ui\\common\\set_up_media_list.py\", line 53, in delete_all_media\n    media_ids, folder_ids = self.get_media_list()\n  File \"D:\\ui\\common\\set_up_media_list.py\", line 34, in get_media_list\n    response = requests.post(url=media_url, headers=headers, data=json.dumps(body))\n  File \"D:\\sw_install\\python-install\\lib\\site-packages\\requests\\api.py\", line 115, in post\n    return request(\"post\", url, data=data, json=json, **kwargs)\n  File \"D:\\sw_install\\python-install\\lib\\site-packages\\requests\\api.py\", line 59, in request\n    return session.request(method=method, url=url, **kwargs)\n  File \"D:\\sw_install\\python-install\\lib\\site-packages\\requests\\sessions.py\", line 587, in request\n    resp = self.send(prep, **send_kwargs)\n  File \"D:\\sw_install\\python-install\\lib\\site-packages\\requests\\sessions.py\", line 701, in send\n    r = adapter.send(request, **kwargs)\n  File \"D:\\sw_install\\python-install\\lib\\site-packages\\requests\\adapters.py\", line 553, in send\n    raise ConnectTimeout(e, request=request)\n"}, "start": 1715758490792, "stop": 1715758511829}], "start": 1715758490792, "stop": 1715758512053}